app/screens/forgot_password/forgot_password.js

```diff
@@ -0,0 +1,200 @@
+                    <FormattedText
+                        style={[style.successTxtColor, style.defaultTopPadding]}
+                        id='password_send.checkInbox'
+                        defaultMessage='Please check your inbox.'
+                    />
+                </View>
+            );
+        } else {
+            passwordFormView = (
+                <View>
+                    <FormattedText
+                        style={[GlobalStyles.subheader, style.defaultTopPadding]}
+                        id='password_send.description'
+                        defaultMessage='To reset your password, enter the email address you used to sign up'
+                    />
+                    <TextInput
+                        ref={this.emailIdRef}
+                        style={GlobalStyles.inputBox}
+                        onChangeText={this.changeEmail}
+                        placeholder={formatMessage({id: 'login.email', defaultMessage: 'Email'})}
+                        autoCorrect={false}
+                        autoCapitalize='none'
+                        keyboardType='email-address'
+                        underlineColorAndroid='transparent'
+                        blurOnSubmit={false}
+                        disableFullscreenUI={true}
+                    />
+                    <Button
+                        containerStyle={GlobalStyles.signupButton}
+                        disabled={!this.state.email}
+                        onPress={this.submitResetPassword}
+                    >
+                        <FormattedText
+                            id='password_send.reset'
+                            defaultMessage='Reset my password'
+                            style={[GlobalStyles.signupButtonText]}
+                        />
+                    </Button>
+                </View>
+            );
+        }
+        return (
+            <View style={style.container}>
+                <StatusBar/>
+                <TouchableWithoutFeedback
+                    onPress={this.blur}
+                >
+                    <View style={style.innerContainer}>
+                        <Image
+                            source={require('assets/images/logo.png')}
+                        />
+                        {displayError}
+                        {passwordFormView}
+                    </View>
+                </TouchableWithoutFeedback>
+            </View>
+        );
+    }
+}
+
+const style = StyleSheet.create({
+    container: {
+        backgroundColor: '#FFFFFF',
+        flex: 1,
+    },
+    innerContainer: {
+        alignItems: 'center',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        paddingHorizontal: 15,
+        paddingVertical: 50,
+    },
+    forgotPasswordBtn: {
+        borderColor: 'transparent',
+        marginTop: 15,
+    },
+    resetSuccessContainer: {
+        marginTop: 15,
+        padding: 10,
+        backgroundColor: '#dff0d8',
```

Comment:
colours and backgrounds are taken from webapp. they are of boot-strap classes and so no theming exists

Reply:
Please verify that it's working as expected when changing user theme.

Reply:
I think we don't support themes on login and password reset pages so this should be fine.